.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - 0.10
+  - 6
+  - 4

```

Comment:
I think there is no need for this... since the test happens in phantomjs anyway.